reflex/utils/types.py

```diff
@@ -164,7 +164,11 @@ def get_attribute_access_type(cls: GenericType, name: str) -> GenericType | None
             prop = descriptor.property
             if not isinstance(prop, Relationship):
                 return None
-            return prop.mapper.class_
+            class_ = prop.mapper.class_
+            if prop.uselist:
+                return list[class_]
```

Comment:
```suggestion
                return List[class_]
```

I think we should use cap `List` to keep py3.8 and py3.9 compatibility, if that works.